src/libraries/System.ComponentModel.Composition/ref/System.ComponentModel.Composition.cs

```diff
@@ -91,11 +91,11 @@ public enum CreationPolicy
     public partial class ExportAttribute : System.Attribute
     {
         public ExportAttribute() { }
-        public ExportAttribute(string contractName) { }
-        public ExportAttribute(string contractName, System.Type contractType) { }
-        public ExportAttribute(System.Type contractType) { }
-        public string ContractName { get { throw null; } }
-        public System.Type ContractType { get { throw null; } }
+        public ExportAttribute(string? contractName) { }
+        public ExportAttribute(string? contractName, System.Type? contractType) { }
```

Comment:
Even if the implementation doesn't currently throw an exception for these being null, is the intention that they be non-null?  Seems like it would be a bug if null were passed.  Can you find any such usage?

Reply:
Didn't found real usage with null, only found unit test with null 😄. 
But the doc says use null or empty string if want to use default, isn't that mean intention was allow null? https://github.com/dotnet/runtime/blob/510113bc7f0a4a505ab0ccb90b0a593c3943baae/src/libraries/System.ComponentModel.Composition/src/System/ComponentModel/Composition/ExportAttribute.cs#L99

Reply:
> isn't that mean intention was allow null?

Yup, ok, thanks.